### PR TITLE
This install method is not officially supported

### PR DESCRIPTION
**Pull request by devinrhode2**
Reading through a few issues on creationix/nvm about updating nvm, you can quickly see it's not officially supported

### CODE REVIEW

**Comment by devinrhode2** [comment]
See https://github.com/creationix/nvm/issues/469

File: Library/Formula/nvm.rb [hunk 1]
```diff
@@ -10,6 +10,8 @@ def install
   end
 
   def caveats; <<-EOS.undent
+      This installation method is not officially supported by nvm maintainers.
+
```
Comment (bfontaine) [inline Library/Formula/nvm.rb]: Could you remove the trailing spaces here? Thanks!

Comment (devinrhode2) [inline Library/Formula/nvm.rb]: no


-Devin <http://zerply.com/DevinRhode2>
<http://zerply.com/devinrhode2>

On Sat, Apr 11, 2015 at 3:04 AM, Baptiste Fontaine <notifications@github.com

**Comment by DomT4** [comment]
It's not supported, but it works. You could apply this caveat in *many* formulae, I don't think it's necessary here.

**Comment by jacknagel** [comment]
Passing on this.

**Pull request closed without merging**